drivers/sensor/sngcja5/Kconfig

```diff
@@ -0,0 +1,11 @@
+# SN-GCJA5 laser particle sensor configuration options
+
+# Copyright (c) 2023 Panasonic Industrial Devices Europe GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+config SNGCJA5
+	bool "SN-GCJA5 laser particle sensor"
+	default y
+	depends on I2C
```

Comment:
```suggestion
	depends on DT_HAS_PANASONIC_SNGCJA5_ENABLED`
	select I2C
```

Reply:
Pushed as requested in latest pull request update.